dll/win32/shell32/dialogs/fprop.cpp

```diff
@@ -104,10 +115,17 @@ SH_ShowPropertiesDialog(LPCWSTR pwszPath, LPCITEMIDLIST pidlFolder, PCUITEMID_CH
     if (PathIsRootW(wszPath))
         return SUCCEEDED(SH_ShowDriveProperties(wszPath, pidlFolder, apidl));
 
+    DWORD style = WS_DISABLED | WS_CLIPSIBLINGS | WS_CAPTION;
+    DWORD exstyle = WS_EX_WINDOWEDGE | WS_EX_APPWINDOW;
+    CStubWindow32 stub;
+    if (!stub.Create(NULL, NULL, NULL, style, exstyle))
+        return E_FAIL;
```

Comment:
Some logging (or a more explicit error code) might be nice.

Reply:
Thanks!